looper/utils.py

```diff
@@ -0,0 +1,157 @@
+""" Helpers without an obvious logical home. """
+
+from collections import defaultdict
+import logging
+import os
+import yaml
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+
+class CommandChecker(object):
```

Comment:
Ah OK, got it. Yeah, it was parallel to `PipelineInterface` before; it just felt like an easy candidate to split from `models` module since--as you said--it's unused. I could see it making sense in a `utils` from an external user's perspective (i.e., if someone wanted to use it for things other than `looper`), but yeah from our within-package perspective I agree--it'd make sense to put it back alongside `PipelineInterface`. I can do that as I bring individual classes out of `models`.